documentation/puppetdb_connection.markdown

```diff
@@ -64,6 +64,21 @@ This setting can let the Puppet master stay partially available during a PuppetD
 
 The default value is false.
 
+## Differences in Puppet Enterprise
+
+### `include_unchanged_resources`
```

Comment:
So to a PE user, all configuration items are equal. They aren't special, FOSS is, by not having/being able to use the feature. See what I mean? We don't want PE users to have to look in two sections, just one if we can. FOSS users can mentally tick off what they lack by a section header marker ... or a bold "Note:" in the config section (although thats less obvious).